### PR TITLE
update dev api documentation

### PR DESCRIPTION
**Pull request by thisab**
**Pull Request check list**

- [ ] Proper tests/regressions included?
- [x] Documentation updated?

**Affected functionality**

**Description of change**

**Which issue this PR fixes**

### CODE REVIEW

File: dev/api/harvester.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        main {
+            flex: 1 0 0;
+            overflow: hidden;
+        }
+    </style>
+</head>
+
+<body>
+    <main role="main">
+        <elements-api apiDescriptionUrl="api_server.yaml" router="hash" />
```
Comment (mchurichi) [inline dev/api/harvester.html:33]:
```suggestion
        <elements-api apiDescriptionUrl="api_harvester.yaml" router="hash" />
```

**Review by mchurichi — CHANGES_REQUESTED** [review]
Thanks for this, Thiago!. I just left a suggestion to point to the right Harvester API spec and we'll be good to go 💯

**Review by jufantozzi — APPROVED** [review]
LGTM 👍

**Review by mchurichi — COMMENTED** [review]
💯

**Review by mchurichi — APPROVED** [review]

**Comment by mchurichi** [comment]
We just need the commits to be signed and we can get this merged 💯

**Review by n-th — APPROVED** [review]

**Comment by mchurichi** [comment]
could you please merge the latest changes from master? I think that would fix the trivy issue, but the yamls have been moved and we'll need to update the paths here

**Comment by thisab** [comment]
i think it would be easier if we dropped this request. So I would pull again and work on the new structure

**Pull request closed without merging**